python/paddle/v2/dataset/common.py

```diff
@@ -32,3 +32,10 @@ def download(url, module_name, md5sum):
             shutil.copyfileobj(r.raw, f)
 
     return filename
+
+
+def dict_add(a_dict, ele):
```

Comment:
No. We don't need to do it.

I tried and it seems that adding to __all__ just helps `from package import *`.